crates/libs/registry/Cargo.toml

```diff
@@ -11,6 +11,10 @@ readme = "readme.md"
 categories = ["os::windows-apis"]
 exclude = ["tests"]
 
+[features]
+default = []
+std = []
```

Comment:
```suggestion
std = ["windows-result/std"]
```

Reply:
Actually, the more I think about feature injection, I think we don't actually need it unless A-->B where A actually uses something from B's std-dependent exterior API.

Reply:
In which case, we should `check` each crate with no default features to make sure we don't miss dependencies

Reply:
* I think `windows-registry` could be entirely `no_std`.
* I could separately add a generated yml workflow for testing no default feature builds.

Reply:
Yeah, I see what you mean.  I'm a bit concerned at the combinatorial explosion in `A{no_std?} --> B{no_std?}` variations, so I'm not sure how to implement this suggestion without things blowing up.  But it's definitely a valid concern...

Reply:
So I think the `windows-registry` crate doesn't need features of its own, depends on the `windows-result` crate with `default-features = false`, and can be `#![no_std]`.

Reply:
Agreed.  I've removed the feature from `windows-registry`; it is now always `no_std` (except in `test` builds).

Reply:
Looks like this has been addressed with some follow-up work. Will merge now to get this batch of changes in.